zarr/determine-chunk-shape.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<blockquote>100MB</blockquote><p>If direct-to-frontend Zarr visualization ever happens, having such large chunks would be a blocker because the frontend would have to download a minimum of one chunk before anything displays.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>